spec/services/alert_spec.rb

```diff
@@ -41,6 +41,14 @@
     )
   end
 
+  it "adds context given as keyword argument" do
+    expect_any_instance_of(Bugsnag::Report).to receive(:add_metadata).with(
+      :data, { data: "ABC123" }
+    )
+
+    Alert.raise("hey", data: "ABC123")
```

Comment:
I find this a bit confusing, using something other `data` in the call to `raise` would make it cleared what's happening, ie:
```suggestion
    expect_any_instance_of(Bugsnag::Report).to receive(:add_metadata).with(
      :error_data, { data: "ABC123" }
    )

    Alert.raise("hey", error_data: "ABC123")

Reply:
Good idea. I'll choose something unique for better readability.